bitnami/pubsub/upstream/kustomization.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - kafka.yaml
+
+patches:
```

Comment:
These don't feel "logging specific"

Reply:
I agree it's not logging specific, template used for this is `clean-upstream-overlay.yaml`